media/css/firefox/new/all-test-variations.less

```diff
@@ -0,0 +1,190 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../sandstone/lib.less";
+
+#fx-modal-wrapper {
+    display: none;
+}
+
+#fx-modal {
+    margin-top: 40px;
+}
+
+#fx-modal-content {
+    .clearfix();
+    background: #fff;
+    border-radius: 6px;
+    color: @textColorPrimary;
+    margin-top: 20px;
+    padding: 40px 20px;
+    text-align: center;
+
+    ul {
+        list-style: none;
+    }
+
+    li {
+        display: inline-block;
+        margin: 0;
+    }
+
+    h3,
+    h4 {
+        color: @textColorPrimary;
+    }
+
+    h3 span {
+        .font-size(18px);
+        display: block;
+        margin-top: 5px;
+        text-align: center;
+    }
+
+    .fx-privacy-link {
+        a:link,
+        a:visited {
+            color: @linkBlue;
+        }
+    }
+}
+
+.fx-modal-section {
+    .border-box();
+    padding: 20px 0;
+    width: 50%;
+}
+
+#fx-modal-secondary-cta {
+    float: left;
+    padding-right: 40px;
+}
+
+#fx-modal-primary-ctas {
+    border-left: 1px solid @textColorLight;
+    float: right;
+    padding-left: 40px;
+}
+
+#fx-modal-mobile-downloads {
+    text-align: center;
+
+    li:first-child {
+        margin-right: 10px;
+    }
+}
+
+#fx-modal-direct-downloads {
+    padding: 10px 30px 0;
+    text-align: left;
+
+    li {
+        .border-box();
+        margin-bottom: 10px;
+        width: 50%;
+
+        a {
+            .font-size(@largeFontSize);
+            background: url('/media/img/firefox/new/horizon/down-arrow.svg') left center no-repeat;
```

Comment:
Nit: now that we've removed the old /new template, this asset could just live at `/media/img/firefox/new/`